Lib/test/test_source_encoding.py

```diff
@@ -13,14 +13,9 @@
 class MiscSourceEncodingTest(unittest.TestCase):
 
     def test_pep263(self):
-        self.assertEqual(
-            "�����".encode("utf-8"),
-            b'\xd0\x9f\xd0\xb8\xd1\x82\xd0\xbe\xd0\xbd'
-        )
-        self.assertEqual(
-            "\�".encode("utf-8"),
-            b'\\\xd0\x9f'
-        )
+        # Test moved to its own file because encoding isn't understood by all editors.
+        # See #96272
+        script_helper.assert_python_ok(findfile("source_encoding_pep263.py"))
```

Comment:
~This can't work as is; `source_encoding_pep263.py` doesn't actually *do* anything.~ After update, it now does something, but only if run without `-O` :)

I think a better solution might be to steal [`ImportTests.test_import_encoded_module`](https://github.com/python/cpython/blob/caa2a9799a47294441e4206037620322eea9ed06/Lib/test/test_imp.py#L69-L73) from `test_imp`.  We don't want to lose that test when `imp` is removed, and these tests are basically duplicates anyway.

Reply:
Ah, that makes sense.  Indeed, `test_import_encoded_modules` koi8-r example seems to test exactly the same thing.